hermes/probe/probe.go

```diff
@@ -16,7 +16,7 @@
 // - Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
 // - Evan Spendlove, GitHub: evanSpendlove.
 //
-// Monitor implements a probe that monitors a storage system using the
+// probe implements a probe that monitors a storage system using the
```

Comment:
This is duplicating the "Package probe" comment. Remove it.

Reply:
Removed.